packages/webapp/src/members/components/members-grid.tsx

```diff
@@ -1,18 +1,20 @@
 import React from "react";
 import Link from "next/link";
 import dayjs from "dayjs";
-import * as localizedFormat from "dayjs/plugin/localizedFormat";
 
 import { atomicAssets, ipfsBaseUrl } from "config";
 import { MemberData } from "../interfaces";
 import { assetToString } from "_app";
 
-dayjs.extend(localizedFormat.default);
-
 interface Props {
     members: MemberData[];
 }
 
+const openInNewTab = (url: string) => {
+    const newWindow = window.open(url, "_blank", "noopener,noreferrer");
+    if (newWindow) newWindow.opener = null;
+};
```

Comment:
oh wow, we really needed this? I wonder why the traditional `<a href={x} target="_blank">` does not work....

Reply:
I tried that but them Chrome/Brave loudly reminded me that you can't have an `<a>` tag within an `<a>` tag. Not valid. So this was the only real way to get the Cmd-click behavior on the card body _and_ on the labels. At least that I could think of before I moved on. 🤷

Reply:
I also tried factoring away from an `<a>` tag at the card level, but again, not the behavior I wanted.

Reply:
Ohhh I see... yeah the badges will overlap with the card footer <a>... :(